jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/sender/PushNotificationSenderEndpoint.java

```diff
@@ -100,9 +101,17 @@
      * @HTTP 401 (Unauthorized) The request requires authentication.
      * @HTTP 404 (Not Found) The requested PushApplication resource does not exist.
      * @RequestHeader aerogear-sender The header to identify the used client. If the header is not present, the standard "user-agent" header is used.
+     *
+     * @param message   message to send
+     * @return          successful string {@code Job submitted}
+     *
+     * @statuscode 202 Indicates the Job has been accepted and is being process by the AeroGear UnifiedPush Server
+     * @statuscode 401 The request requires authentication
+     * @statuscode 404 The requested PushApplication resource does not exist
      */
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
+    @ReturnType("java.lang.String")
     public Response send(final UnifiedPushMessage message, @Context HttpServletRequest request) {
```

Comment:
this generates more doc than we want, in this case.

For instance the UnifiedPushMessage has `ipAddress` for internal tracking - that is not exposed, see:
https://aerogear.org/docs/specs/aerogear-unifiedpush-rest/sender/index.html